### ISSUE TITLE
Enhance Visual Studio extension templates

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#6647]
## What would you like to be added:

Adhere to `NetAnalyzers` rules.

## Why is this needed:

Better code-quality, less noise if a user wants to enable `NetAnalyzers`.

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [x] Solution Templates

**Comment by Youssef1313** [comment]
I'll look into this.

**Comment by xperiandri** [comment]
Do you want some specific .editorconfig setup?

**Comment by Youssef1313** [comment]
Nope. I just enabled all rules from NetAnalyzers and fixed what seems reasonable (the template itself won't have NetAnalyzers enabled).
